ament_copyright/ament_copyright/parser.py

```diff
@@ -54,14 +55,6 @@ def read(self):
     def parse(self):
         raise NotImplemented()
 
-    def identify_license(self, content, license_part):
-        for name, license in get_licenses().items():
-            if content is not None and getattr(license, license_part) == content:
-                self.license_identifier = name
-                break
-        else:
-            self.license_identifier = UNKNOWN_IDENTIFIER
-
```

Comment:
So, I wasn't entirely sure what to do about this.  We need to do something slightly different in this method in the subclass SourceDescriptor than we do in ContributingDescriptor and LicenseDescriptor.  Thus, my options seemed to be:

* Override it in SourceDescriptor
* Add in some special-case flag for when we wanted to do additional substitutions
* Copy the code

Given that it's really only 4 lines of code, I opted for the last one.  However, I'm open to other suggestions.